plugins/mds.py

```diff
@@ -376,3 +377,16 @@ def calculation_task(self, db_id: int) -> str:
         )
 
     return "Result stored in file"
+
+
+def _get_dim_attributes(dim):
```

Comment:
We may want to introduce helper methods for this into the plugin runner some time.